Anatomy of a Script File.ipynb

```diff
@@ -5,39 +5,59 @@
    "metadata": {},
```

Comment:
<p>I would just say "Because HOOMD-Blue has a Python interface", since technically it still running C++ under the hood  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PTC-CMC/cybercamp/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Good point...we want to make sure we are being clear.